src/routes/routes.js

```diff
@@ -8,5 +8,6 @@ const router = express.Router();
 router.post('/api/v1/users', Validation.signupValidation, UserAuth.signup);
 router.post('/api/v1/send-email', Validation.emailValidation, UserAuth.emailSender);
 router.post('/api/v1/reset-password/:token', verifyToken, Validation.passwordValidation, UserAuth.resetPassword);
+router.post('/api/v1/login', Validation.signInValidation, UserAuth.login);
```

Comment:
be consistent with terms used `signIn` vs `login`

Reply:
Done